public/documentation/embedding_crosswalk/crosswalk_aar.md

```diff
@@ -114,6 +114,8 @@ Then install it into the local Maven repository:
     }
     ```
 
+    Using mavenLocal() instead of remote repo url to refer local aar.
+
```

Comment:
No, maybe we should add those scripts under "Using mavenLocal() instead of remote repo url to refer local aar."
Repositories {
     mavenLocal()
}